hicommonwealth/commonwealth
# Kill ERC20 and ERC721

## Link to Issue
Closes: #4461 

## Description of Changes
- Deleted all ERC20 and ERC721 code from chain-events

## Test Plan
- Run `yarn start-rmq`
- Navigate to `packages/commonwealth` and run `yarn reset-db && yarn load-db`
- Start the CW app server with run `yarn start`
- In a new terminal, navigate to `packages/chain-events` and run `yarn start-all`
- Ensure the chain-event logs appear normal (no errors -> osmosis may produce 429 errors due to rate limiting)
- Test loading proposals on various communities (e.g. dydx, osmosis, save) -> **This currently does not work due to existing proposal loading issues**

## Deployment Plan

1. 

## Other Considerations

-
